src/utils/data.ts

```diff
@@ -117,3 +111,5 @@ export type ContentTreeArray = {
   summary?: string | undefined;
   body: string;
 };
+
+export const LANGUAGECODES = ["zh", "es", "pt"];
```

Comment:
This is already defined elsewhere
- https://github.com/bitcointranscripts/registry/blob/f5099125a620300b401bd6ba6a6d1fc6590b8443/src/utils/index.ts#L55
- https://github.com/bitcointranscripts/registry/blob/f5099125a620300b401bd6ba6a6d1fc6590b8443/src/config/index.ts#L1


Let's try to keep a single source of truth for stuff. This should also be included in our contribution guidelines @Emmanuel-Develops . I know that this is a small issue, but it points to a broader need to ensure we reference and build on what’s already in the codebase.

As it is, when in the future we need to make a change to supported language we will need to make changes in multiple files, which increases the chances of forgetting something and introducing an issue.

Reply:
Good points made and I will make necessary corrections.

I already defined this earlier as it was non-existent in the codebase the time this PR was first created, I think it was introduced in later merged PRs. 

Will make updates. Thanks for the review.